b2sdk/b2http.py

```diff
@@ -120,6 +120,9 @@ def _translate_and_retry(fcn, try_count, post_params=None):
         except B2Error as e:
             if not e.should_retry_http():
                 raise
+            if hasattr(e, 'retry_after_seconds') and e.retry_after_seconds:
+                time.sleep(e.retry_after_seconds)
```

Comment:
it will sleep here AND two lines below. Twice. I think the sleep below should be in `else:`

Reply:
I have a continue there but I can add else.

Reply:
oh, I missed it, sorry!